e2e/consul/input/on_update_check_restart.nomad

```diff
@@ -0,0 +1,88 @@
+job "test" {
```

Comment:
E2E tests need to filter their target by kernel so they don't break if they get deployed onto Windows. (Same for all 3 jobs.)